pkg/metrics/metrics.go

```diff
@@ -216,6 +216,12 @@ const (
 	LabelAddressType          = "address_type"
 	LabelAddressTypePrimary   = "primary"
 	LabelAddressTypeSecondary = "secondary"
+
+	// LabelPolicyNamespace is the label for the namespace of the policy that references a CIDRGroup
+	LabelPolicyNamespace = "policyNamespace"
```

Comment:
For that matter, I'm not seeing these labels getting used anywhere anyway. If we're not using them, then this code should be removed. Or they should be used when observing the metrics.

Reply:
It's a leftover from the previous revision, I'll remove it.